cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/task/LoadMonitorTaskRunner.java

```diff
@@ -277,6 +282,12 @@ public void resumeSampling() {
     }
   }
 
+  /**
+   * Allow tasks to know if another thread, e.g. executor, is waiting on sampling to pause.
+   */
+  public boolean awaitingPauseSampling() {
+    return _awaitPauseSampling;
```

Comment:
Nit: Let's keep the getter name and the variable name consistent -- i.e.
`_awaitPauseSampling` -> `_awaitingPauseSampling`.